src/main/java/org/jenkinsci/plugins/stashNotifier/StashNotifier.java

```diff
@@ -604,6 +626,7 @@ public boolean configure(
             prependParentProjectKey = formData.getBoolean("prependParentProjectKey");
 
 			disableInprogressNotification = formData.getBoolean("disableInprogressNotification");
+			buildState = formData.getString("buildState");
```

Comment:
As mentioned in #108, this causes a crash in Jenkins global configuration page. I'm not familiar with Jenkins plugin development, but I'm not sure you have to query the buildState here as this method seems to be used only to keep global configurations.